pkg/object/mqttproxy/storage.go

```diff
@@ -84,15 +97,31 @@ func (m *mockStorage) put(key, value string) error {
 func (m *mockStorage) delete(key string) error {
 	m.mu.Lock()
 	delete(m.store, key)
+	flag := atomic.LoadInt32(&m.watchFlag)
```

Comment:
be careful to use atomic function, most of usage like this is wrong, I gave same comments in the TCP pr.

Reply:
this one is actually fine...... when broker call `watchDelete` at start. it will set this flag true. then when delete the message will set to broker....